.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Are we tracking vscode recommendations per module?

Reply:
I think this is PDK-ified now?